src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,256 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.model.User;
+import org.junit.jupiter.api.BeforeAll;
```

Comment:
Consider using `@BeforeEach` instead of `@BeforeAll` for initializing the `RegistrationServiceImpl` instance. `@BeforeAll` is used for static setup that happens once before all tests run, while `@BeforeEach` is used for initializing test state before each test. Since `registrationService` is not a resource-intensive operation and you want to ensure a fresh state for each test, `@BeforeEach` may be more appropriate here.